lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -69,12 +74,25 @@ def main(argv=None):
                              'This cycletime is in the format '
                              'YYYYMMDDTHHMMZ.')
     # Filepaths for historic and truth data.
-    parser.add_argument('historic_filepath', metavar='HISTORIC_FILEPATH',
+    parser.add_argument('--historic_filepath', metavar='HISTORIC_FILEPATH',
```

Comment:
It's not necessarily clear that these options are mutually exclusive, you want either (--historic_filepath and --truth_filepath) or --combined_filepath.

Reply:
Yes, I've extended the help within the CLI to try to be clear about which combinations are valid.